src/Serilog.Sinks.Console/Serilog.Sinks.Console.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A Serilog sink that writes log events to the console/terminal.</Description>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
soz missed that one. I was looking for `project.json` :stuck_out_tongue_winking_eye: